src/main/java/com/factory/repository/UsersRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.factory.repository;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import com.factory.model.Users;
+
+@Repository
+public interface UsersRepository extends CrudRepository<Users, Long> {
+	
+	Users findByUserName(@Param("userName") String userName);
```

Comment:
Lets try not to use queries with spring instead write actual queries. Please look into JDBC template